e2e/test/scripts/infrastructure-linode/0.1.0/clusterclass-kubeadm.yaml

```diff
@@ -0,0 +1,397 @@
+    apt-get install -y kubelet=$PATCH_VERSION* kubeadm=$PATCH_VERSION* kubectl=$PATCH_VERSION* containerd
+    apt-mark hold kubelet kubeadm kubectl containerd
+    modprobe overlay
+    modprobe br_netfilter
+    sysctl --system
+    sed -i '/swap/d' /etc/fstab
+    swapoff -a
+  sysctl-k8s.conf: |
+    net.bridge.bridge-nf-call-iptables  = 1
+    net.bridge.bridge-nf-call-ip6tables = 1
+    net.ipv4.ip_forward                 = 1
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: linode-${CLUSTER_NAME}-crs-0
+stringData:
+  linode-token-region.yaml: |-
+    kind: Secret
+    apiVersion: v1
+    metadata:
+      name: linode-token-region
+      namespace: kube-system
+    stringData:
+      apiToken: ${LINODE_TOKEN}
+      region: ${LINODE_REGION}
+type: addons.cluster.x-k8s.io/resource-set
+---
+apiVersion: addons.cluster.x-k8s.io/v1alpha1
+kind: HelmChartProxy
+metadata:
+  name: cilium
+spec:
+  chartName: cilium
+  clusterSelector:
+    matchLabels:
+      cni: cilium
+  namespace: kube-system
+  options:
+    timeout: 5m
+    wait: true
+    waitForJobs: true
+  repoURL: https://helm.cilium.io/
+  valuesTemplate: |
+    ipam:
+      mode: kubernetes
+    k8s:
+      requireIPv4PodCIDR: true
+    hubble:
+      relay:
+        enabled: true
+      ui:
+        enabled: true
+  version: ${CILIUM_VERSION:=1.15.0}
+---
+apiVersion: addons.cluster.x-k8s.io/v1alpha1
+kind: HelmChartProxy
+metadata:
+  name: linode-cloud-controller-manager
+spec:
+  chartName: ccm-linode
+  clusterSelector:
+    matchLabels:
+      ccm: linode
+  namespace: kube-system
+  options:
+    timeout: 5m
+    wait: true
+    waitForJobs: true
+  repoURL: https://linode.github.io/linode-cloud-controller-manager/
+  valuesTemplate: |
+    secretRef:
+      name: "linode-token-region"
+    image:
+      pullPolicy: IfNotPresent
+  version: ${LINODE_CCM_VERSION:=v0.3.24}
+---
+apiVersion: addons.cluster.x-k8s.io/v1beta1
+kind: ClusterResourceSet
+metadata:
+  name: ${CLUSTER_NAME}-crs-0
+spec:
+  clusterSelector:
+    matchLabels:
+      crs: ${CLUSTER_NAME}-crs
+  resources:
+  - kind: Secret
+    name: linode-${CLUSTER_NAME}-crs-0
+  strategy: ApplyOnce
+---
+apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+kind: KubeadmConfigTemplate
+metadata:
+  name: kubeadm-worker
+spec:
+  template:
+    spec:
+      files:
+      - contentFrom:
+          secret:
+            key: containerd-config.toml
+            name: common-init-files
+        path: /etc/containerd/config.toml
+      - contentFrom:
+          secret:
+            key: k8s-modules.conf
+            name: common-init-files
+        path: /etc/modules-load.d/k8s.conf
+      - contentFrom:
+          secret:
+            key: sysctl-k8s.conf
+            name: common-init-files
+        path: /etc/sysctl.d/k8s.conf
+      - contentFrom:
+          secret:
+            key: kubeadm-pre-init.sh
+            name: common-init-files
+        path: /kubeadm-pre-init.sh
+        permissions: "0500"
```

Comment:
Don't need execute perms?

Reply:
@avestuk the manifests were generated by the project itself. So nothing to do here.